src/keras_helper.py

```diff
@@ -31,51 +31,71 @@ def __init__(self):
 
     def add_conv_layer(self, img_size=(32, 32), img_channels=3):
         self.classifier.add(BatchNormalization(input_shape=(*img_size, img_channels)))
+
+        self.classifier.add(Conv2D(32, (3, 3), padding='same', activation='relu'))
         self.classifier.add(Conv2D(32, (3, 3), activation='relu'))
-        self.classifier.add(MaxPooling2D(pool_size=(2, 2)))
+        self.classifier.add(MaxPooling2D(pool_size=2))
```

Comment:
This is actually the same thing but it's just a matter of preferences. I wanted to keep a tuple to not forget that it's the 2x2 patch but you can keep it like that if you want :)